File: BackEnd/src/main/java/org/example/dto/ClienteDTO.java
package org.example.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ClienteDTO {
    private Long idCliente;
    private String nombre;
    private String email;
    private Integer cantidadVehiculos;
}

File: Android/app/src/main/java/com/example/autolog/core/data/remote/dto/cliente/ClienteDTO.kt
package com.example.autolog.core.data.remote.dto.cliente

import com.google.gson.annotations.SerializedName

data class ClienteDTO(
    @SerializedName("idCliente")
    val idCliente: Long,
    
    @SerializedName("nombre")
    val nombre: String,
    
    @SerializedName("email")
    val email: String,
    
    @SerializedName("cantidadVehiculos")
    val cantidadVehiculos: Int
)
